bigtop-packages/src/common/hive/do-component-build

```diff
@@ -36,12 +36,12 @@ HIVE_MAVEN_OPTS=" -Dhbase.version=$HBASE_VERSION \
 # for non-x86 platforms
 if [ $HOSTTYPE = "powerpc64le" ] ; then
   mvn install:install-file -DgroupId=com.google.protobuf -DartifactId=protoc -Dversion=2.5.0 \
-    -Dclassifier=linux-ppcle_64 -Dpackaging=exe -Dfile=/usr/local/bin/protoc
+    -Dclassifier=linux-ppcle_64 -Dpackaging=exe -Dfile=/usr/local/protobuf-2.5.0/bin/protoc
 elif [ $HOSTTYPE = "aarch64" ] ; then
   mvn install:install-file -DgroupId=com.google.protobuf -DartifactId=protoc -Dversion=2.5.0 \
-    -Dclassifier=linux-aarch_64 -Dpackaging=exe -Dfile=/usr/local/bin/protoc
+    -Dclassifier=linux-aarch_64 -Dpackaging=exe -Dfile=/usr/local/protobuf-2.5.0/bin/protoc
 fi
-export MAVEN_OPTS="${MAVEN_OPTS} -Xmx1500m -Xms1500m"
+export MAVEN_OPTS="${MAVEN_OPTS} -Xmx2500m -Xms2500m"
```

Comment:
How about removing `-Xms1500m` to make JVM allocate heap on demand? We can not expect much gain by allocating maximum heap from the beginning for short term task like building products. I got no OOM with current settings on my x86_64 environment.

Reply:
It makes sense.  Also tested it on my Arm64 environment.

Reply:
While `-Xmx2500m` might be still useful to allocate enough heap on machine with memory less than 5GB (because the default maximum heap of JVM is capped at half of the machine's memory IIRC), we can not practically use such a small machine for Bigtop development. I think dropping `-Xmx` too should be ok.